lib/beamchmark.ex

```diff
@@ -61,7 +61,7 @@ defmodule Beamchmark do
   @doc """
   Runs scenario and benchmarks EVM performance.
 
-  Subsequent invocation of this function will also compare results with the previous ones.
+  If `compare?` option equals `true`, invocation of this function will also compare new measurements with the previous.
```

Comment:
It is `true` by default, from `Beamchmark.options_t()` description:
![image](https://user-images.githubusercontent.com/31112335/156355992-6e3c3420-323d-4024-a0ea-88042b185fc8.png)
I wanted to avoid repeating this in `Beamchmark.run/2` doc, do you think we should mention this also there?

Reply:
Ahh, my bad :man_facepalming: It's ok